### PR TITLE
Replace copyright notices to dunnhumby Germany GmbH

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**
Due to organizational changes in sociomantic labs GmbH the copyright owner must be updated to the legal entity "dunnhumby Germany GmbH".

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -173,6 +169,4 @@ the `Tsunami programme`_ created by `sociomantic labs GmbH`_.
 .. _All: https://github.com/sociomantic-tsunami/libdrizzle-redux/releases/
 .. _here: https://github.com/sociomantic-tsunami/neptune/blob/master/doc/library-user.rst
 .. _compiling.rst: https://github.com/andreas-bok-sociomantic/libdrizzle-redux/blob/v5.4.x/docs/compiling.rst
-.. _Tsunami programme: https://github.com/sociomantic-tsunami
-.. _sociomantic labs GmbH: https://www.sociomantic.com
 .. _Neptune: https://github.com/sociomantic-tsunami/neptune/blob/master/doc/library-user.rst#contributing-to-a-neptune-versioned-library
```
Comment (ben-palmer-sociomantic) [inline README.rst]: Not strictly related to this PR but the neptune links are broken. Should be:
`https://github.com/sociomantic-tsunami/neptune/blob/v0.x.x/doc/library-user.rst`
and
`https://github.com/sociomantic-tsunami/neptune/blob/v0.x.x/doc/library-user.rst#contributing-to-a-neptune-versioned-library`

**Review by ben-palmer-sociomantic — COMMENTED** [review]
Just one comment not strictly related to the PR. Fix if you like.

File: README.rst (hunk 1 above)
Comment (andreas-bok-sociomantic) [reply README.rst]: :scream: I will make a separate PR to fix them

**Comment by ben-palmer-sociomantic** [comment]
Need to rebase on master now :)

**Review by leandro-lucarella-sociomantic — APPROVED** [review]
LGTM

**Comment by andreas-bok-sociomantic** [comment]
Fixed conflict

**Comment by ben-palmer-sociomantic** [comment]
LGTM

**Pull request merged**